sentry-android-replay/src/main/java/io/sentry/android/replay/util/FixedWindowCallback.java

```diff
@@ -0,0 +1,254 @@
+/**
+ * Adapted from https://github.com/square/curtains/tree/v1.2.5
+ *
+ * <p>Copyright 2021 Square Inc.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.sentry.android.replay.util;
+
+import android.annotation.SuppressLint;
+import android.view.ActionMode;
+import android.view.KeyEvent;
+import android.view.KeyboardShortcutGroup;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.MotionEvent;
+import android.view.SearchEvent;
+import android.view.View;
+import android.view.Window;
+import android.view.WindowManager;
+import android.view.accessibility.AccessibilityEvent;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Implementation of Window.Callback that updates the signature of {@link #onMenuOpened(int, Menu)}
+ * to change the menu param from non null to nullable to avoid runtime null check crashes. Issue:
+ * https://issuetracker.google.com/issues/188568911
+ */
+public class FixedWindowCallback implements Window.Callback {
```

Comment:
I know we have WindowCallbackAdapter, but I don't wanna depend on sentry-android-core just for that single class, so I've added another one (enhanced, as it fixes an NPE actually)